### DISCUSSION TITLE
Shrink matrix jobs on pull requests

### DISCUSSION THREAD

**Discussion by PaulLesur** [community/community#26406] [Actions]
<p>Hi !<br>
I don’t know if I am in the right category. But it could be good to shrink the matrix jobs in a pull request’s checks. For example, I have a project that runs twenty runners that run my tests in a matrix, I would like to be able to shrink these jobs in the pull request checks, so that I will only see a single job and not twenty jobs.<br>
I don’t know if I explained my request correctly…</p>

**Comment by Simran-B** [comment, marked as answer]
<p>Sounds similar to <a href="https://github.community/t/feat-group-the-checks-in-the-pr-by-each-yaml-file/139485" class="inline-onebox">Feat: group the checks in the PR by each yaml file</a></p>
